cmd/abad/cli/cli_test.go

```diff
@@ -0,0 +1,53 @@
+package cli_test
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/NeowayLabs/abad/cmd/abad/cli"
+	"github.com/madlambda/spells/assert"
+)
+
+var trim = strings.TrimSpace
+
+func TestCli(t *testing.T) {
+	for _, tc := range []struct {
+		in  string
+		out string
+		err error
+	}{
+		{
+			in:  "0",
+			out: "0",
+		},
+		{
+			in:  "0xff",
+			out: "255",
+		},
+		{
+			in:  "1.0e1",
+			out: "10",
+		},
+		{
+			in:  "1e-10",
+			out: "0.0000000001",
+		},
+		{
+			in:  "1e10",
+			out: "10000000000",
+		},
+	} {
+		var inb bytes.Buffer
+		var outb bytes.Buffer
+		cli := cli.NewCli("test.js", &inb, &outb)
+
+		_, err := inb.Write([]byte(tc.in + "\n"))
```

Comment:
You can use a [WriteString](https://golang.org/pkg/bytes/#Buffer.WriteString) to avoid the cast